xarray/tests/test_backends.py

```diff
@@ -1767,6 +1771,77 @@ def test_setncattr_string(self) -> None:
                 assert_array_equal(one_element_list_of_strings, totest.attrs["bar"])
                 assert one_string == totest.attrs["baz"]
 
+    @pytest.mark.parametrize(
+        "compression",
+        [
+            None,
+            "zlib",
+            "szip",
+            "zstd",
+            "blosc_lz",
+            "blosc_lz4",
+            "blosc_lz4hc",
+            "blosc_zlib",
+            "blosc_zstd",
+        ],
+    )
+    def test_compression_encoding(self, compression: str | None) -> None:
+        if self.nc4_version < Version("1.6.2"):
+            pytest.skip(
```

Comment:
Better to skip that at test collection time. See `xarray/tests/__init__.py` for examples on how to create such a decorator.